src/containers/Tenant/Diagnostics/DetailedOverview/DetailedOverview.scss

```diff
@@ -3,15 +3,19 @@ $section-title-line-height: 24px;
 
 .kv-detailed-overview {
     display: flex;
-    gap: 20px;
 
+    width: 100%;
     height: 100%;
+    gap: 20px;
 
     &__section {
         display: flex;
         overflow-x: hidden;
         flex: 0 0 calc(50% - 10px);
         flex-direction: column;
+
+        min-width: 300px;
```

Comment:
In tenant info the only content that could overflow - tenant name, for other items 300px is enough.